examples/rosettacode/semordnilap.nit

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Simple Rosettacode task for the detection of semordnilaps in a text file
+#
+# SEE: http://rosettacode.org/wiki/Semordnilap
+module semordnilap
+
+if args.length < 1 then
+	print "Usage: ./semordnilap file"
+	exit(1)
+end
+
+var fr = new FileReader.open(args[0])
+var words = new HashSet[String]
+
+for i in fr.each_line do words.add i.trim
```

Comment:
also name a variable to avoid using a semantic-less `args[0]`.

~~~nit
var filename = args[0]
~~~